### PR TITLE
PHP-Fusion (CMS)

### PR DESCRIPTION
**Pull request by dwblmo**
Installation of PHP-Fusion, a CMS, on U7

### CODE REVIEW

File: source/guide_php-fusion.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+.. sidebar:: Logo
+
+  .. image:: _static/images/php-fusion.png
+      :align: center
+
+##########
+PHP-Fusion
+##########
+
+PHP-Fusion_ is a CMS written in PHP and distributed under the GN AGPL v3 licence.
```
Comment (SalocinHB) [inline source/guide_php-fusion.rst]: please provide URLs for your links. Also, you probably meant the _GNU AGPL_

File: source/guide_php-fusion.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PHP_
```
Comment (SalocinHB) [inline source/guide_php-fusion.rst]: missing URL

File: source/guide_php-fusion.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1:
```
Comment (SalocinHB) [inline source/guide_php-fusion.rst]: missing URL

File: source/guide_php-fusion.rst [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+------
+
+Your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+First get the PHP-Fusion source code from Github_:
```
Comment (SalocinHB) [inline source/guide_php-fusion.rst]: missing URL

File: source/guide_php-fusion.rst [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  Receiving objects: 100% (88715/88715), 111.94 MiB | 14.20 MiB/s, done.
+  Resolving deltas: 100% (61743/61743), done.
+  [isabell@stardust ~]$
+
+  
+Finishing installation
+======================
+
+Point your browser to your domain, for example https://isabell.uber.space,
+and follow the installation instructions.
```
Comment (SalocinHB) [inline source/guide_php-fusion.rst]: I only get an error message

`Fatal error: Uncaught Error: Class 'Defender' not found in /var/www/virtual/testnico/html/includes/dynamics/includes/form_select.php:344 Stack trace: #0 /var/www/virtual/testnico/html/includes/classes/PHPFusion/Installer/Steps/Introduction.inc(84): form_select('localeset', '', 'English', Array) #1 /var/www/virtual/testnico/html/includes/classes/PHPFusion/Installer/Steps/Introduction.inc(29): PHPFusion\Steps\InstallerIntroduction->__Index() #2 /var/www/virtual/testnico/html/includes/classes/PHPFusion/Installer/Install.core.inc(351): PHPFusion\Steps\InstallerIntroduction->__view() #3 /var/www/virtual/testnico/html/includes/classes/PHPFusion/Installer/Install.core.inc(330): PHPFusion\Installer\Install_Core->get_InstallerContent() #4 /var/www/virtual/testnico/html/install.php(22): PHPFusion\Installer\Install_Core->install_phpfusion() #5 {main} thrown in /var/www/virtual/testnico/html/includes/dynamics/includes/form_select.php on line 344`

Comment (dwblmo) [reply source/guide_php-fusion.rst]: Yes, that's a bit of shame, that the developers of PHP-Fusion just don't fix this issue, although it was opened and closed with the argument "it's development" -> https://github.com/php-fusion/PHP-Fusion/issues/2212

I think I'll rewrite the whole installation routine using the latest stable version.

**Pull request merged**